### PR TITLE
Fix/htj2k and keymodifier

### PR DESCRIPTION
**Pull request by wayfarer3130**
Fix HTJ2K loading in the streaming volume image loader.
Also, fix the key modifier so that it actually works on mouse event information

### CODE REVIEW

File: packages/tools/src/eventDispatchers/shared/getMouseModifier.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { KeyboardBindings as kb } from '../../enums';
+
+/**
+ * Gets the mouse modifier key from a mouse event.
+ * Supports Shift, Ctrl, Alt, in singly and in combinations of 2
+ * Supports Meta singly.
+ */
+const getMouseModifierKey = (evt) =>
```
Comment (sedghi) [inline packages/tools/src/eventDispatchers/shared/getMouseModifier.ts:8]: The logic below is too cumbersome and I don't understand what it is doing. Can you please add more docs? So the evt is mouseEvent, it sees if the shiftKey is pressed with mouse? while ctrl and shiftCtrol is there OR alt or shiftAlt, I'm just lost

**Review by sedghi — COMMENTED** [review]
Can you please add more docs to the getMouseModifier, I don't understand a bit of that function. Otherwise looks good

File: packages/tools/src/eventDispatchers/shared/getMouseModifier.ts (hunk 1 above)
Comment (wayfarer3130) [reply packages/tools/src/eventDispatchers/shared/getMouseModifier.ts]: Redid the logic - it is the same logic, just easier to understand as I broke it out.

**Review by sedghi — APPROVED** [review]
Looks great

**Pull request merged**